gravity.sh

```diff
@@ -367,7 +367,7 @@ gravity_Schwarzschild() {
     # Determine if file has read permissions, as download might have failed
     if [[ -r "${i}" ]]; then
       # Remove windows CRs from file, and append into $matterAndLight
```

Comment:
Could you please modify the comment to read:
`# Remove windows CRs from file, convert list to lower case, and append into $matterAndLight`

Reply:
Okay, done